--- origsrc/xmp-4.0.6/src/sound_win32.c	2013-04-06 08:38:02.000000000 -0500
+++ src/xmp-4.0.6/src/sound_win32.c	2013-11-11 22:29:57.697410600 -0600
@@ -58,8 +58,8 @@ static void show_error(int res)
 	fprintf(stderr, "Error: %s", msg);
 }
 
-static void CALLBACK wave_callback(HWAVEOUT hwo, UINT uMsg, DWORD dwInstance,
-				   DWORD dwParam1, DWORD dwParam2)
+static void CALLBACK wave_callback(HWAVEOUT hwo, UINT uMsg, DWORD_PTR dwInstance,
+				   DWORD_PTR dwParam1, DWORD_PTR dwParam2)
 {
 	if (uMsg == WOM_DONE) {
         	freebuffer++;
@@ -94,7 +94,7 @@ static int init(struct options *options)
 	    wfe.wBitsPerSample / 8;
 	wfe.nBlockAlign = wfe.nChannels * wfe.wBitsPerSample / 8;
 
-	res = waveOutOpen(&hwaveout, WAVE_MAPPER, &wfe, (DWORD) wave_callback,
+	res = waveOutOpen(&hwaveout, WAVE_MAPPER, &wfe, (DWORD_PTR) wave_callback,
 			  0, CALLBACK_FUNCTION);
 
 	if (res != MMSYSERR_NOERROR) {
